gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/main/java/io/gravitee/am/gateway/handler/scim/mapper/UserMapper.java

```diff
@@ -193,6 +197,10 @@ public static io.gravitee.am.model.User convert(User scimUser) {
 
         // set additional information
         if (scimUser instanceof GraviteeUser graviteeUser && graviteeUser.getAdditionalInformation() != null) {
+            var lastPasswordReset = graviteeUser.getAdditionalInformation().get(LAST_PASSWORD_RESET_KEY);
+            if (lastPasswordReset instanceof Long) {
+                user.setLastPasswordReset(Date.from(Instant.ofEpochMilli((Long) lastPasswordReset)));
```

Comment:
**question:** should we manage Long or should we manage String format as it is the case the Metadata https://datatracker.ietf.org/doc/html/rfc7643. I'm ok with the long as it is the format used in the MAPI but I raise this point as the dates defiend for the metadata are using String ...

Reply:
@tcompiegne I think this question is for you :)

Reply:
IMO we should be align of the same format across the SCIM API (so we should go with the String)... 
Long is more handy than string but having two date formats inside the same API is confusing IMO.